src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -87,31 +24,35 @@
                   separator=";"
                   tableName="jhi_user"
                   usePreparedStatements="true">
+            <column name="id" type="uuid"/>
```

Comment:
it's important to be exhaustive with the types, else Liquibase will try to get a snapshot of the database and it's not supported yet by the Neo4j extension